src/main/resources/assets/immersiveengineering/manual/en_us/balloon.txt

```diff
@@ -3,6 +3,6 @@ Balloon
 <&recipe>Balloons are sacks of fabric with a heatsource below to keep them afloat. They will emit light, but despite the torch, they are not a fire hazard.
 Balloons can be placed like normal blocks, or when no block is targeted, they will be placed in the air infront of you.
 Similar to <link;metalconstruction;Structural Connectors;connector_recipes> you can attach Hemp Rope and Steel Wire to the balloon.
-Sneak-clicking allows you to set a height offset for the balloon, allowing you to place it up to 5 blocks above the target.
+Sneak-clicking while placing allows you to set a height offset for the balloon, allowing you to place it up to 5 blocks above the target. Sneaking and right-clicking will pick the balloon up.
```

Comment:
I think I'd prefer "Sneaking and right-clicking on a balloon will let you pick it up."